src/styles/reset.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding-inline-start: 0;
+}
```

Comment:
Setting the margin of every element to 0 can potentially impact performance by increasing the time it takes for the browser to render the page.

Reply:
Also, reset.scss has to be in utils.